Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -1187,6 +1184,12 @@ internal protected void OnTargetEvent (TargetEventArgs args)
 			if (args.Thread != null && args.IsStopEvent)
 				activeThread = args.Thread;
 
+			if (stepTimer != null && (HasExited || args.IsStopEvent)) {
+				stepTimer.Stop (true);
+				stepTimer.Dispose ();
+				stepTimer = null;
+			}
```

Comment:
Looking at this now we no longer know if there was an error in the stepping? Before if an exception was thrown the timer.Success flag would not be set to true.

Reply:
True, but I doubt it is a very common scenario and if we care about that, we should provide it another way.